loader_hub/apify/actor/README.md

```diff
@@ -0,0 +1,48 @@
+# Apify Actor Loader
+
+[Apify](https://apify.com/) is a cloud platform for web scraping and data extraction,
+which provides an [ecosystem](https://apify.com/store) of more than a thousand
+ready-made apps called *Actors* for various scraping, crawling, and extraction use cases.
+
+This loader runs a specific Actor and loads its results.
+
+## Usage
+
+In this example, we’ll use the [Website Content Crawler](https://apify.com/apify/website-content-crawler) Actor,
+which can deeply crawl websites such as documentation, knowledge bases, help centers,
+or blogs, and extract text content from the web pages. 
+The extracted text then can be fed to a vector index in order to answer questions from it.
+
+To use this loader, you need to have a (free) Apify account 
+and provide your [Apify API token](https://console.apify.com/account/integrations) as the `APIFY_API_TOKEN`
+environment variable. Note that the loader requires
+the [apify-client](https://pypi.org/project/apify-client/) PyPI package.
```

Comment:
The `apify-client` is installed on its own apparently: https://github.com/emptycrown/llama-hub#step-1-create-a-new-directory